src/function/algebra/simplify/simplifyCore.js

```diff
@@ -70,14 +67,31 @@ export const createSimplifyCore = /* #__PURE__ */ factory(name, dependencies, ({
    *
    * @param {Node} node
    *     The expression to be simplified
+   * @param {Object} options
+   *     Simplification options, as per simplify()
    */
-  function simplifyCore (node) {
-    if (isOperatorNode(node) && node.isUnary()) {
-      const a0 = simplifyCore(node.args[0])
-
-      if (node.op === '+') { // unary plus
-        return a0
+  function simplifyCore (node, options) {
+    const context = options ? options.context : undefined
+    if (hasProperty(node, 'trivial', context)) {
+      // This node does nothing if it has only one argument, so if so,
+      // return that argument simplified
+      if (isFunctionNode(node) && node.args.length === 1) {
+        return simplifyCore(node.args[0], options)
+      }
+      // For other node types, we try the generic methods
+      let simpChild = false
+      let childCount = 0
+      node.forEach(c => {
+        if (++childCount === 1) {
```

Comment:
I'm personally not a fan of using `++` and `===` in a one liner, I always have to think twice to understand what is happening (and it is easy to introduce an off-by-one bug). Not a big deal, but if ok with you I would prefer splitting it in two lines: first do the increment, next do the equality check.

Reply:
Split